Makefile

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020 Red Hat, Inc.
+
```

Comment:
Need to add `# Copyright Contributors to the Open Cluster Management project` to all files

Reply:
Done